cross_list_courses/templates/add_new.html

```diff
@@ -74,11 +74,14 @@ <h3>
                             {{ mapping.primary_school_id | upper }}
                             <b>
                             {% if mapping.primary_school_id == 'hls' or mapping.primary_school_id == 'ext' %}
-                                {{ mapping.primary_registrar_code }}
+                                {{ mapping.primary_registrar_code|truncatechars:20  }}
```

Comment:
Should this and primary_short_title be 40 char?

Reply:
Yes, fixed the secondary and not primary. thanks!